src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -149,10 +146,10 @@ private List<ValidationResult> validateCategoryId(final String dataFieldId,
             return Collections.emptyList();
         }
 
-        List<ValidationResult> validationResults =
```

Comment:
Removing the textValidator.validate() removes Max and Min validation along with Regex. Is this expected?

Reply:
Those checks work better for fields in the CaseFields tab. You can specify the min, max and regex for a field. For 
 Complex Types it's for for the complex type itself than the items inside it.

Reply:
So for an item like category_id in a Document{} a Min, Max isn't needed. Unless it's made a requirement.